packages/st2/in-requirements.txt

```diff
@@ -19,3 +19,4 @@ stackstorm-runner-orquesta
 stackstorm-runner-python
 stackstorm-runner-remote
 stackstorm-runner-winrm
+wheel
```

Comment:
Small code comment why this change was included would help future generations.

Reply:
I was able to remove this based on better understanding the `EL8` python situation and it has been fixed in the dockerfile

Reply:
running a local build to confirm